modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/env ruby
 
+def create_machines_hash
+  machines_names_hash = Hash.new
+
+  `/usr/local/bin/govuk_node_list --with-puppet-class`.each_line do |machine|
+    machine_fqn_name = machine.split(":")
```

Comment:
I think the following works in Ruby:

```ruby
    domain_name, puppet_class = machine.chomp.split(":")
    machine_names_hash[domain_name] = puppet_class
```

The benefit being that the two values being moved around have more meaningful names.